src/parachain/system.ts

```diff
@@ -12,6 +13,12 @@ export interface SystemAPI {
      * @returns The current active block number being processed.
      */
     getCurrentActiveBlockNumber(): Promise<number>;
+
+    /**
+     * On every new parachain block, call the callback function with the ID of the expired request.
```

Comment:
This comment should be changed as it's outdated

Reply:
What should it be changed to? As far as I understand it, the subscription does exactly that, call the callback on every new finalised header. Does that not happen on every block?

> it's outdated

I just wrote it about an hour ago, so it could easily be completely wrong but I don't think it's "outdated" haha

Reply:
```suggestion
     * On every new parachain block, call the callback function with the new block header
```
Sorry about the wording - by outdated I meant that it might have been copied over from another TS-Doc comment

Reply:
Ooh, right. You're absolutely correct, thanks.